includes/admin/class-wp-job-manager-settings.php

```diff
@@ -322,7 +322,7 @@ public function output() {
 								break;
 								case "password" :
 
-									?><input id="setting-<?php echo $option['name']; ?>" class="regular-text" type="password" name="<?php echo $option['name']; ?>" value="<?php esc_attr_e( $value ); ?>" <?php echo implode( ' ', $attributes ); ?> <?php echo $placeholder; ?> /><?php
+									?><input id="setting-<?php echo $option['name']; ?>" class="regular-text" type="password" name="<?php echo $option['name']; ?>" value="<?php echo esc_attr( $value ); ?>" <?php echo implode( ' ', $attributes ); ?> <?php echo $placeholder; ?> /><?php
```

Comment:
Just a note, not 100% if this is the case here (may need to monitor support after) but plugins like WPML can translate settings values and this would show the localised version depending on admin access. 

On the flip side, there are other ways to translate WPML settings using their XML files, so this fix is probably fine. Just thought I'd mention it.